### PR TITLE
no signature check

### PR DESCRIPTION
**Pull request by calvinmetcalf**
removed the signature check as this prevents you from using the  private keys that come in the .json key bundles, plus the verifying didn't do much as we'd know about the error soon enough.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ var auth = function(key, options, callback) {
     };
     jwtHeaderB64 = base64urlEncode(JSON.stringify(jwtHeader));
     iat = Math.floor(new Date().getTime() / 1000);
+    console.log(iat);
```
Comment (jcblw) [inline index.js]: can we just remove this log.

**Comment by jcblw** [comment]
looks good just that one minor revision :+1:

**Comment by calvinmetcalf** [comment]
whoops put that in there when I was debuging a problem for far too long
that turned out to be me passing the option iis instead of iss, fixed

**Comment by jcblw** [comment]
cool, probably publish it later today

**Pull request merged**